docs/useRequireAccess.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+title: "useRequireAccess"
+---
+
+# `useRequireAccess`
+
+This hook calls the `authProvider.canAccess()` method on mount for a provided resource and action (and optionally a record). If users don't have access, it will redirect them to the [`/unauthorized`](./Admin.md#unauthorized) page. If `authProvider.canAccess()` throws an error, it will redirect them to the [`/authentication-error`](./Admin.md#authenticationerror) page.
```

Comment:
Now I think the names of the routes are wrong. 

The first one should be `/access-denied`.

The second could be `/authorization-error`, but we may keep `/authentication-error` as a generic page for all errors related to the `authProvider`.

Reply:
So `accessDenied` for the prop?